src/regexp.rs

```diff
@@ -0,0 +1,73 @@
+use crate::support::int;
```

Comment:
Missing copyright header

Reply:
none of the other files i looked at have it 🤔